app/assets/javascripts/discourse/tests/fixtures/session-fixtures.js

```diff
@@ -33,7 +33,71 @@ export default {
       timezone: "Australia/Brisbane",
       skip_new_user_tips: false,
       can_review: true,
-      ignored_users: []
+      ignored_users: [],
+      groups: [
```

Comment:
😿

Reply:
I realised I can probably at least cut this down to the essentials. It more reflects reality -- in most cases every user will have _at least_ TL0 and TL1 in this array, so it makes sense to include it in the fixture.